gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -262,11 +246,24 @@ def main(args):  # noqa: D103
                 overwrite=args.overwrite,
             )
 
-        # TODO: Need to use annotate_subpop_meta before subpop assignment
         if args.assign_subpops:
-            raise NotImplementedError(
-                "Sub-population assignment is not currently implemented."
+            logger.info("Assigning subpops...")
+            joint_pca_ht, joint_pca_fit = assign_pops(
+                min_prob=args.min_prob,  # How to decide on this number? Should withhold a certain percent and make a PR curve gs://gnomad-julia/gnomad_v4/pca_with_ccdg_gnomad_ukb_variants.ipynb
+                include_unreleasable_samples=False,
+                max_mislabeled_training_samples=args.max_mislabeled_training_samples,  # How to decide on this number?
+                pcs=args.pcs,
+                withhold_prop=args.withhold_prop,
+                pop=pop,
+                high_quality=high_quality,
+            )
+
+            joint_pca_ht = joint_pca_ht.checkpoint(
+                assigned_subpops_path(pop),
+                overwrite=args.overwrite,
+                _read_if_exists=not args.overwrite,
             )
```

Comment:
No used after so write instead of checkpoint
```suggestion
            joint_pca_ht.write(assigned_subpops_path(pop), overwrite=args.overwrite)
```